Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM alpine:3.11
+
+RUN apk add curl
+
+RUN apk add python3
+
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3 -
+
+COPY . .
+
+RUN pip3 install flask
+
+RUN source $HOME/.poetry/env && poetry update && poetry install
+
+RUN pip3 install gunicorn
+
+RUN pip3 install requests
+
+FROM base as production
```

Comment:
I don't think we've defined a `base` image yet - we need to add `as base` to the first line so that it knows which layers that refers to

Reply:
ah makes sense, ty